app/controllers/StatsController.php

```diff
@@ -101,35 +109,32 @@ function students() {
             $lastname = isset($name[1]) ? $name[1] : '';
           }
         }
-        echo "$firstname,$lastname,$email\n";
-        // $name = isset($student['name']) ? $student['name'] : '';
-        // $last_name = preg_replace('#.*\s([\w-]*)$#', '$1', $name);
-        // $first_name = trim(preg_replace('#'.$last_name.'#', '', $name ));
-        // if (strlen($first_name) == 0) {
-        //   $first_name = $last_name;
-        //   $last_name = "";
-        // }
-        // echo "$first_name,$last_name,$email\n";
-      }
-      echo '</textarea>';
+        $allStudents[] = ["firstname" => $firstname, "lastname" => $lastname, "email" => $email];
+
+      $studentsArray['studentsConfirmedEmails'] = $confirmedEmails;
+      $studentsArray['studentsUnconfirmedEmails'] = $unconfirmedEmails;
+      $studentsArray['allStudents'] = $allStudents;
+      return $studentsArray;
     }
+  }
+
+    /*
+    * Returns recruiters who are missing jobs
+    */
     function missingrecruiter() {
       global $MStats;
+      $missingRecruiterArray = [];
 
+      $mrArray = [];
       $mr = $MStats->getJobsMissingRecruiter();
-      echo '<br />Jobs nonexistent recruiter: '.count($mr).'<br />
-        <textarea style="width:800px; height: 200px;">';
-      foreach ($mr as $job) {
-        $id = $job['_id'];
-        $company = $job['company'];
-        $recruiter = $job['recruiter'];
-        echo "$id - c: $company, r: $recruiter\n";
-      }
-      echo '</textarea>';
+
+      $missingRecruiterArray["missingRecruiters"] = $mr;
```

Comment:
You can simply do:
`$missingRecruiterArray = ['missingRecruiters' => $mr];`
and not have to do the `$missingRecruiterArray = [];` above.